src/appleseed.studio/mainwindow/rendering/lightpathswidget.cpp

```diff
@@ -52,7 +55,7 @@
 // Standard headers.
 #include <algorithm>
 #include <cmath>
-#include <string>
+#include <iostream>
```

Comment:
If you need to log stuff, use our logger. iostream should only be used for debug purpose.

Reply:
Ah yes, I was using for debugging and forgot to remove it